kolibri/plugins/html5_viewer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -123,6 +123,9 @@
       this.hashi.onStateUpdate(data => {
         this.$emit('updateContentState', data);
       });
+      this.hashi.on('navigateto', message => {
+        this.$emit('navigateToRegularContext', message);
```

Comment:
I think just `navigateTo` seems fine here as an event - it's up to the context embedding the content renderer to decide what to do with that.